snapshot/snapshot.go

```diff
@@ -359,7 +351,7 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 		return nil, err
 	}
 
-	logger.Debugf("[Prepare] snapshot with labels %v", info.Labels)
+	logger.Infof("[Prepare] snapshot with labels %v", info.Labels)
```

Comment:
The labels may contain sensitive information. So we should avoid print it as INFO level

Reply:
Fixed